<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>S_Prediction</class>
 <widget class="QMainWindow" name="S_Prediction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="p_spinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>43</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="d_spinBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>43</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="q_spinBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>43</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QComboBox" name="c_nc_comboBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>94</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>c</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nc</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ARIMA_pushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>분석하기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>p 값(자기상관)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>d 값(차분)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>q 값(이동평균)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>c : 상수포함</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nc : 상수미포함</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="OpenBtn"/>
    <addaction name="separator"/>
    <addaction name="SAVE"/>
    <addaction name="separator"/>
    <addaction name="CloseBtn"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="OpenBtn">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>OPEN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="SAVE">
   <property name="text">
    <string>SAVE</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="CloseBtn">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
